glyphs-reader/src/font.rs

```diff
@@ -1971,7 +1973,22 @@ impl Instance {
             value
                 .width_class
                 .as_ref()
-                .map(|v| f64::from_str(v).unwrap())
+                .map(|v| {
+                    match i32::from_str(v).unwrap() {
+                        1 => 50.0,  // Ultra-condensed
+                        2 => 62.5,  // Extra-condensed
+                        3 => 75.0,  // Condensed
+                        4 => 87.5,  // Semi-condensed
+                        5 => 100.0, // Medium (normal)
+                        6 => 112.5, // Semi-expanded
+                        7 => 125.0, // Expanded
+                        8 => 150.0, // Extra-expanded
+                        9 => 200.0, // Ultra-expanded
```

Comment:
Optional, it would be nicer to enumerate the OS/2 values, parse str to that, and then ask the enum for the width %

Reply:
I did that here [190c059](https://github.com/googlefonts/fontc/pull/921/commits/190c059ff4cd7a8bfb238dae3b7fb77b774d8c45), but I'm not sure I like it better than, the previous code was more concise

Reply:
actually, I'm going to move this enum WidthClass to fontdrasil so that I can use in the FE as well as the BE, I need to map back from a percent value to u16 in order to fix https://github.com/googlefonts/fontc/issues/904